policyengine_us/variables/gov/ed/pell_grant/efc/pell_grant_simplified_formula_applies.py

```diff
@@ -15,3 +15,6 @@ def formula(person, period, parameters):
         total_benefits = add(tax_unit, period, p.benefits)
         has_benefits = np.any(total_benefits > 0)
         return income_eligible & has_benefits
+
+    def formula_2024(person, period, parameters):
```

Comment:
can we rather make a bool parameter and use it as an if statement here? 
Something like 
```
if p.simplified_formula_applies.yaml:
    tax_unit = person.tax_unit
    head_income = tax_unit("pell_grant_primary_income", period)
    p = parameters(period).gov.ed.pell_grant.efc.simplified
    income_eligible = head_income < p.income_limit
    total_benefits = add(tax_unit, period, p.benefits)
    has_benefits = np.any(total_benefits > 0)
    return income_eligible & has_benefits
else:
    False
```